tdrs-frontend/src/components/Request/Request.jsx

```diff
@@ -18,11 +18,7 @@ function Request() {
         <p className="margin-top-1 margin-bottom-4">
           An administrator will be in touch soon to confirm your access!
         </p>
-        <Button
-          className="width-full request-access-button"
-          type="button"
-          onClick={signOut}
-        >
+        <Button className="width-full" type="button" onClick={signOut}>
```

Comment:
I was able to remove the extra class name because the buttons are globally styled now.